docs/algorithms/kem/ml_kem.yml

```diff
@@ -58,6 +58,18 @@ parameter-sets:
     no-secret-dependent-branching-claimed: true
     no-secret-dependent-branching-checked-by-valgrind: true
     large-stack-usage: false
+  - upstream: primary-upstream
+    upstream-id: aarch64
+    supported-platforms:
+    - architecture: ARM64_V8
+      operating_systems:
+      - Linux
+      - Darwin
+    common-crypto:
+    - SHA3: liboqs
+    no-secret-dependent-branching-claimed: true
+    no-secret-dependent-branching-checked-by-valgrind: false
```

Comment:
Is it really the case that we're not doing Valgrind constant-time testing of this implementation in liboqs? If so, why?

Reply:
I can't comment on what tests libOQS itself has, but mlkem-native does have valgrind tests, including for absence of secret-dependent division.

Reply:
This is only for the Arm64 implementations.
@bhess said that liboqs currently does not have valgrind tests for Arm64.

Reply:
> liboqs currently does not have valgrind tests for Arm64.

This is (already/)so documented for those algs that have `aarch64` optimizations (Kyber, Dilithium, Falcon).

Reply:
My mistake, thanks for the reminder Michael.